src/set_theory/cardinal/basic.lean

```diff
@@ -615,45 +614,35 @@ theorem bdd_above_iff_small (s : set cardinal.{u}) : bdd_above s ↔ small.{u} s
   { simp_rw [subtype.val_eq_coe, equiv.symm_apply_apply], refl }
 end⟩
 
-/-- The indexed supremum of cardinals is the smallest cardinal above
-  everything in the family. -/
-def sup {ι : Type u} (f : ι → cardinal.{max u v}) : cardinal :=
-supr f
```

Comment:
It's perhaps worth noting in the PR description that changing from `sup` to `supr` has relaxed the requirement on universe arguments.